### ISSUE TITLE
Add question: An accessibility issue is only occuring with one specific setup (browser, screen reader + browser, OS, etc) - is this a fail?

### ISSUE THREAD

**Issue by MaximPerry** [cenw-wscoe/WCAG-in-Plain-Language#32]

**Comment by MaximPerry** [comment]
My proposed answer:

No. 
If the accessibility issue only occurs using a specific browser, screen reader, operating system, etc. That falls in the category of a software bug coming from those technologies, not a problem in the webpage's code.
